apps/style/common.scss

```diff
@@ -107,6 +107,10 @@ html[dir='rtl'] #visualizationColumn {
   }
 }
 
+#pause-button:focus:not(:focus-visible) {
```

Comment:
Open question here - if we want to reuse this focus-visibility in other settings on other buttons, I'm wondering if we should add this to something more generic?

Reply:
I can make a class for it instead of using an ID